internal/controllers/nmc_reconciler.go

```diff
@@ -867,6 +877,21 @@ func setWorkerContainerArgs(pod *v1.Pod, args []string) error {
 	return nil
 }
 
+func setWorkerPrivileged(pod *v1.Pod, privileged bool) error {
```

Comment:
we are already setting security context in the baseWorkerPod function. Can we just add an if clause there and set the priviliged flag?

Reply:
We could, but then we would keep adding parameters to that function. If you insist, I can change it.